examples/all-clusters-app/p6/BUILD.gn

```diff
@@ -54,6 +54,30 @@ p6_sdk_sources("all_clusters_app_sdk_sources") {
   public_configs = [ "${chip_root}/third_party/p6:p6_sdk_config" ]
 }
 
+static_library("static-supported-modes-manager") {
```

Comment:
Again, why the separate library instead of adding to sources for `p6_executable("clusters_app")`?